lib/src/widgets/score_list/score_list.dart

```diff
@@ -28,9 +33,9 @@ class ScoreList extends StatelessWidget {
             return ScoreListElement(
               index: index,
               scoreDirection: ScoreDirection.rise,
-              username: 'nancy',
-              imgUrl: 'assets/mock/pictures/profile-4.jpg',
-              score: 1337,
+              username: getUsernameByUid(userScoreList[index + 3].key),
```

Comment:
anstelle dieser aufwändigen Logik könntest du auch einmal am Anfang der build Methode in der ListView den user holen. Den Index bekommst du ja. Wenn in der userScoreList dein user nicht mit drin ist, wäre es ratsam, das zusammen zu packen. 

ich würde eine klasse machen die den score gesamt beinhaltet, den Rang und den User